src/stock.rs

```diff
@@ -217,8 +217,13 @@ pub enum StockMessage {
     #[strum(props(fallback = "You are invited to a video chat, click %1$s to join."))]
     VideochatInviteMsgBody = 83,
 
-    #[strum(props(fallback = "Configuration failed. Error: “%1$s”"))]
+    #[strum(props(fallback = "Error:\n\n“%1$s”"))]
```

Comment:
I do not think we should show anything with "Configuration" to the user.
I will need to do another pr to really change this.

Reply:
i am late to the party, i commented on https://github.com/deltachat/deltachat-android/pull/1630#issuecomment-697280364